app/helpers/templates_helper.rb

```diff
@@ -103,4 +103,10 @@ def resource_value_options(resource_type)
   def resource_permission(resource_type)
     "view_#{resource_type.demodulize.underscore.pluralize}".to_sym
   end
+
+  def template_class(template)
```

Comment:
Could we be a little bit more specific please? helper method name clash is hard to debug.
Maybe something like `template_class_name` would be at least bit more descriptive... :shrug: